Assets/Scripts/Player/PlayerLookController.cs

```diff
@@ -8,7 +8,9 @@ public class PlayerLookController : MonoBehaviour
     
 	private Transform playerBody;
 	private float pitch;
+
     private bool isShopping = false;
+    private bool isViewingInventory = false;
```

Comment:
You didn't pull my master change I assume, cuz this stuff was overhauled to use `PlayerLookController.locked`

Reply:
ya saw that in your master - prefer it too, but was running into those merge conflicts. let's do that part together when we merge